src/preamble.js

```diff
@@ -933,9 +943,9 @@ function updateGlobalBufferViews() {
   Module['HEAPF64'] = HEAPF64 = new Float64Array(buffer);
 }
 
-var STATIC_BASE = 0, STATICTOP = 0, staticSealed = false; // static area
```

Comment:
I see, thanks.

I think we do still want these inits, as in your code fragment. It matters for the JS engine: if it sees the whole life of the variable can only contain int values, it's better than if it can be undefined as well. When undefined is in the mix its harder to optimize.

Also please add a comment explaining that it would otherwise be initted by the pthreads code.